Makefile

```diff
@@ -122,9 +122,9 @@ infra.clean:
 	@cd dev-infrastructure && DEPLOY_ENV=$(DEPLOY_ENV) make clean
 .PHONY: infra.clean
 
-infra.observability:
-	cd observability && KUBECONFIG="$$(cd ../dev-infrastructure && make -s svc.aks.kubeconfigfile)" make
-.PHONY: infra.observability
+infra.tracing:
```

Comment:
Should we keep observability then  here too?

cc @simonpasquier

Reply:
I would be against, cause it's deploying tracing and not observability